### PR TITLE
Use baked default skybox

### PR DESCRIPTION
**Pull request by Atlante45**
Bake default skybox to save ~1s on startup time.

## Test Plan:
1) Launch interface
2) Make sure you see the default skybox

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1438,15 +1438,18 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
...
 
-    QString skyboxUrl { PathUtils::resourcesPath() + "images/Default-Sky-9-cubemap.jpg" };
-    QString skyboxAmbientUrl { PathUtils::resourcesPath() + "images/Default-Sky-9-ambient.jpg" };
+        auto skyboxUrl = PathUtils::resourcesPath().toStdString() + "images/Default-Sky-9-cubemap.ktx";
+        auto skyboxAmbientUrl = PathUtils::resourcesPath().toStdString() + "images/Default-Sky-9-ambient.ktx";
 
-    _defaultSkyboxTexture = textureCache->getImageTexture(skyboxUrl, image::TextureUsage::CUBE_TEXTURE, { { "generateIrradiance", false } });
-    _defaultSkyboxAmbientTexture = textureCache->getImageTexture(skyboxAmbientUrl, image::TextureUsage::CUBE_TEXTURE, { { "generateIrradiance", true } });
+        _defaultSkyboxTexture = gpu::Texture::unserialize(skyboxUrl);
+        _defaultSkyboxAmbientTexture = gpu::Texture::unserialize(skyboxAmbientUrl);
```
Comment (huffman) [inline interface/src/Application.cpp]: Can we ditch the ambient texture and use the `images/Default-Sky-9-cubemap.ktx` texture for both now? IIRC, we only had the lower-resolution ambient texture for minimizing the time it took to process irradiance, and presumably those irradiance values are now in the baked ktx.

Comment (Atlante45) [inline interface/src/Application.cpp]: Oh really? Yeah for sure.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5208daaaeb1ab284b3819bf4202aece861eb4164 available at: https://gist.github.com/70e7ddd670cee6e729163cb2575c27a8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-5208daa.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-5208daa.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 79d53d92a1e07de9d991f24fbb46bc7c894b8d3b available at: https://gist.github.com/70e7ddd670cee6e729163cb2575c27a8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-79d53d9.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-79d53d9.dmg

**Comment by PhilipRosedale** [comment]
QA FAIL:  I see all black screen, no skybox, in both desktop and HMD

**Comment by Atlante45** [comment]
Dev build runs fine, so I'm assuming that's a packaging issue. Looking into it now.

**Comment by murillodigital** [comment]
test this please

**Comment by Atlante45** [comment]
test this please

**Comment by RyanDowne** [comment]
Is this dev approved now?

**Comment by Atlante45** [comment]
Waiting on the build, Jenkins is having issues posting the executables.
I just need @huffman to look at the change I added.

File: libraries/shared/src/shared/Storage.cpp [hunk 2]
```diff
@@ -90,3 +90,31 @@ FileStorage::~FileStorage() {
...
+    if (_file.open(QFile::ReadWrite)) {
+        _mapped = _file.map(0, _file.size());
+        if (_mapped) {
+            _valid = true;
+            _hasWriteAccess = true;
+        } else {
+            qCWarning(storagelogging) << "Failed to map file " << _file.fileName();
+        }
+    } else {
+        qCWarning(storagelogging) << "Failed to open file " << _file.fileName();
```
Comment (huffman) [inline libraries/shared/src/shared/Storage.cpp]: Should we throw an exception in these cases? This will almost definitely cause issues later.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e0863aa50f3509b3102f33d694f7d129bc33fbcb available at: https://gist.github.com/70e7ddd670cee6e729163cb2575c27a8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-e0863aa.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-e0863aa.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6378b96b4c9eca5683e44726aedede1b293e48d1 available at: https://gist.github.com/70e7ddd670cee6e729163cb2575c27a8<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-6378b96.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10437-6378b96.dmg

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

Default skybox is black for around 4 seconds as it loads but then turns into to the normal skybox.

I'll let you decide if that is a pass, it doesn't turn black after it's loaded though so I'd assume so.

Let me know if there is anything else I can do/provide.

**Comment by Atlante45** [comment]
@YuppyPlays Those 4 seconds are just startup time before the default skybox is loaded.
I have another PR almost ready that addresses this.

Thanks for the QA!

**Comment by YuppyPlays** [comment]
No worries :)

**Comment by RyanDowne** [comment]
For me, I launch interface and see the default skybox, then the sky goes black and then the skybox texture loads. This feels like when the skybox texture is loading, it decides to show the color selected in the zone's "skybox color"

**Comment by Atlante45** [comment]
@RyanDowne It's probably the ambient map; since that's what we compute the irradiance from, we tend to make them smaller from what I understand. That means we download and draw them first, before we get the albedo of the skybox.

Not related to this PR specifically.

**Pull request merged**